Sources/HummingbirdPostgres/Migrations.swift

```diff
@@ -66,8 +66,18 @@ public actor PostgresMigrations {
     ///   - logger: Logger to use
     ///   - dryRun: Should migrations actually be applied, or should we just report what would be applied and reverted
 
-    public func apply(client: PostgresClient, groups: [PostgresMigrationGroup] = [], logger: Logger, dryRun: Bool) async throws {
-        try await self.migrate(client: client, migrations: self.migrations, groups: groups, logger: logger, completeMigrations: true, dryRun: dryRun)
+    public func apply(
+        client: PostgresClient,
+        groups: [PostgresMigrationGroup] = [],
+        logger: Logger,
+        dryRun: Bool
+    ) async throws {
+        // wait a small period to ensure the PostgresClient has started up
+        try await Task.sleep(for: .microseconds(100))
```

Comment:
Isn't this something that's considered a bug/missing feature in PostgresNIO? This is super awkward

Reply:
@fabianfett